src/py/flwr/cli/build.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2024 Flower Labs GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Flower command line interface `build` command."""
+
+import hashlib
+import os
+import zipfile
+from pathlib import Path
+from typing import Optional
+
+import jwt
+import pathspec
+import typer
+from typing_extensions import Annotated
+
+from .config_utils import validate_project_dir
+from .utils import is_valid_project_name
+
+
+# pylint: disable=too-many-locals
+def build(
+    directory: Annotated[
+        Optional[Path],
+        typer.Option(help="The Flower project directory to bundle into a FAB"),
+    ] = None,
+    signed: Annotated[bool, typer.Option(help="Flag to sign the FAB")] = False,
```

Comment:
Let's use imperative form here (`signed` -> `sign`):

```suggestion
    sign: Annotated[bool, typer.Option(help="Flag to sign the FAB")] = False,
```